content/docs/cucumber/api.md

```diff
@@ -848,6 +848,22 @@ The **Command-Line Interface Runner (CLI Runner)** is an executable Java class t
 ```
 java io.cucumber.core.cli.Main
 ```
+Note that you will need to add the following jars to your classpath, in addition to the location of your compiled .class files:
+
+* `cucumber-core-4.8.0.jar`
```

Comment:
Let's use the version(s) from versions.yaml; we have the cucumber-jvm version in there; not sure about the gherkin etc.) We should either add them there (without breaking things for other programming languages!) or make this description more generic, stating only which jars to add to the classpath and where/how to find the correct versions)